stable/aws-for-fluent-bit/templates/configmap.yaml

```diff
@@ -127,6 +127,29 @@ data:
         {{- end }}
 {{- end }}
 
+{{- if .Values.elasticsearch.enabled }}
+    [OUTPUT]
+        Name            es
+        Match           {{ .Values.elasticsearch.match }}
+        AWS_Region      {{ .Values.elasticsearch.region }}
+        AWS_Auth        {{ .Values.elasticsearch.aws_auth }}
+        {{- if .Values.elasticsearch.host }}
+        host            {{ .Values.elasticsearch.host }}
+        {{- end }}
+        {{- if .Values.elasticsearch.port }}
+        Port            {{ .Values.elasticsearch.port }}
+        {{- end }}
+        {{- if .Values.elasticsearch.tls }}
+        TLS             {{ .Values.elasticsearch.tls }}
+        {{- end }}
+        {{- if .Values.elasticsearch.retry_limit }}
+        Retry_Limit     {{ .Values.elasticsearch.retry_limit }}
+        {{- end }}
+        {{- if .Values.elasticsearch.timeKeyFormat }}
+        time_key_format {{ .Values.elasticsearch.timeKeyFormat }}
```

Comment:
and this is "Time_Key_Format" not "time_key_format"

Reply:
I removed it as I didn't test the whole LogStash feature